configurations/memory_basic.json

```diff
@@ -1,5 +1,11 @@
 {
-  "environment": "Reacher-v2",
+  "environment": "ReacherMemory-v0",
+  "environment_attributes": {
+    "observation_frames": 20,
+    "memory_frames": 20,
+    "action_frames": 50,
+    "observation_mask": [4, 5, 8, 9, 10]
```

Comment:
Bei einer Liste im Config File muss man aufpassen da diese als die Design Space interpretiert werden könnte. ich würde diese Maske gar nicht im Configfile unterbringen, sondern direkt fest im Reacher Memory Experiment verankern. Es macht aus meiner Sicht keinen Sinn eine andere Observation Maske für dieses Experiment festzulegen.

Reply:
Nutzen wir das mit den Listen als Design Space hier im neuen Repo?

> Es macht aus meiner Sicht keinen Sinn eine andere Observation Maske für dieses Experiment festzulegen.

Ja stimmt, ich hardcode das ins Experiment rein :+1:

Reply:
ich denke bei listen in der config, könnte man das so machen, dass die doppelt verpackt werden müssen im design space. Den Code  zum  samplen aus dem design space müssten man dann nur dahin ändern, dass man abfragt ob der parent schon eine liste war, bevor man den child knoten als liste behandelt. 

Also wenn meine normale observation mask so aussieht 

    "observation_mask": [4, 5, 8, 9, 10]

dann sieht das im design space so aus

    "observation_mask": [[4, 5, 8, 9, 10]]

und wenn ich im design space die auswahl zwischen 3 listen haben will, dann sieht es so aus

    "observation_mask": [[1,2,3,4],[3,4,5,6,7],[6,7,8,9,10]]